### ISSUE TITLE
Network loss backward problems ?

### ISSUE THREAD

**Issue by Wenchao-Du** [NVIDIA/pix2pixHD#94]
generally during the gan training,  the loss of discriptor is first to backward, then loss of generator start to backward, however the order is inverse according to the code in the train.py file.  why does it ? and i found the gan loss in the code used the L1/l2 loss functon as cross enctry loss function, is it related the loss function? can you give your explainations about it ? thank you!
